### PR TITLE
Add delayed fees to release notes

### PR DESCRIPTION
**Pull request by thomash-acinq**
Add release notes for #2201

### CODE REVIEW

**Comment by pm47** [comment]
> @pm47 Should we set eclair.relay.fees.enforcement-delay to 0 by default so that there is no change in the default behaviour?

No I think the new default of 10 min is okay, but I would state the value explicitly in the release note.

File: docs/release-notes/eclair-vnext.md [hunk 1]
```diff
@@ -12,7 +12,12 @@
 
 ### Miscellaneous improvements and bug fixes
 
-<insert changes>
+#### Delay enforcement of new channel fees
+
+When updating the relay fees for a channel, eclair can now continue accepting to relay payments using the old fee even if they would be rejected with the new fee.
```
Comment (remyers) [inline docs/release-notes/eclair-vnext.md:17]:
```suggestion
When updating the relay fees for a channel, eclair can now continue accepting relay payments using the old fee even if they would be rejected with the new fee.
```

Comment (remyers) [reply docs/release-notes/eclair-vnext.md:17]: typo

**Review by remyers — APPROVED** [review]
Looks good. I think it's ok to default to 10 minutes for updated nodes - Rusty's research shows that on average that's how long it takes for updates to propagate so it should be safe for planned updates. 

Only in the case where someone makes an incorrect change and does not know how to invalidate it immediately is there a problem - so it's good that this is discussed in the release notes.

**Pull request merged**